app/assets/javascripts/discourse/app/controllers/preferences/interface.js

```diff
@@ -418,5 +418,12 @@ export default Controller.extend({
         lightStylesheet.remove();
       }
     },
+
+    resetSeenPopups(event) {
+      event.preventDefault();
```

Comment:
We can also kind of tell that we're using the wrong element here by having to do this.